com.dropbox.Client.desktop

```diff
@@ -0,0 +1,36 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Dropbox
+Name[ar]=Dropbox
+Name[bn]=ড্রপবক্স
+Name[de]=Dropbox
+Name[es]=Dropbox
+Name[fr]=Dropbox
+Name[hi]=ड्रॉपबॉक्स
+Name[id]=Dropbox
+Name[pt_BR]=Dropbox
+Name[ro]=Dropbox
+Name[th]=Dropbox
+Name[vi]=Dropbox
+Name[zh_CN]=Dropbox
+Name[zh_TW]=Dropbox
+Comment=Access your files from any computer
+Comment[ar]=الوصول إلى ملفاتك من أي كمبيوتر
+Comment[bn]=যে কোন কম্পিউটার থেকে আপনার ফাইলসমূহে প্রবেশ করুন
+Comment[de]=Von jedem Computer aus Zugriff auf die eigenen Dateien
+Comment[es]=Accede a tus documentos desde cualquier computadora
+Comment[fr]=Accédez à vos fichiers depuis n'importe quel ordinateur
+Comment[hi]=किसी भी कम्प्यूटर से अपने फाइलों तक पहुंचें 
+Comment[id]=Mengakses berkas dari komputer mana saja
+Comment[pt_BR]=Acesse seus arquivos à partir de qualquer computador
+Comment[ro]=Acceseaza fisierele tale de pe orice computer
+Comment[th]=เข้าถึงไฟล์ของคุณจากเครื่องไหนก็ได้
+Comment[vi]=Truy cập các tệp từ bất kỳ máy tính nào
+Comment[zh_CN]=从任何计算机访问您的文件
+Comment[zh_TW]=從任何電腦存取您的檔案
+Exec=/app/bin/eos-dropbox-app
+Icon=com.dropbox.Client
+Categories=Utility;
+X-Endless-LaunchMaximized=false
+X-Flatpak=com.dropbox.Client
```

Comment:
This can be removed, it's added by Flatpak on export.

Reply:
Ok